### ISSUE TITLE
Issue with vim-pasta

### ISSUE THREAD

**Issue by oldtimeguitarguy** [tacahiroy/ctrlp-funky#103]
Here's what happens:

0. Installed plugins: ctrlp, ctrl-funky, vim-pasta

1. Open vim

2. Use ctrlp to search for something with the letter "p" in it. Everything is good.

3. Use ctrlp-funky to search for something with the letter "p" in it. The following error occurs:

```
Error detected while processing function <SNR>38_NormalPasta:
line    2:
E21: Cannot make changes, 'modifiable' is off
Press ENTER or type command to continue
```

4. Repeat  step 2 - suddenly we get the same error as step 3

**Comment by tacahiroy** [comment]
hmm - I don't reproduce this. Can you provide CtrlP, Funky and vim-pasta configurations if you still use these plugins pls.

**Comment by jcanseco** [comment]
I had the same issue, but in my case, vim-pasta complained even when CtrlP-Funky was disabled. It complained whenever I pressed 'p' while on the CtrlP prompt. 

I managed to fix it by adding the following line to my `.vimrc`:
```
let g:pasta_disabled_filetypes = ['ctrlp']
```

Source: someone had the same problem [when pressing 'p' on NERDTree](https://www.bountysource.com/issues/20446351-vim-pasta-interferes-with-nerdtree-p-go-to-parent-node).
